### ISSUE TITLE
Research materials

### ISSUE THREAD

**Issue by adheesh-k** [CSAllenISD/2023-ISP-AutomatedWindowShades#8]

**Comment by AlanRoybal** [comment]
I think we should look into making out product an attachment rather than a permanent fixture.

**Comment by HengYiLee** [comment]
The NECESSARY materials found to be required:
- Raspberry Pi
- Plastic or Wood (Must look further into pricing)
- Sautering Kit
- Illuminance Meter

Other materials may be found to be required for the foreseeable future.

**Comment by adheesh-k** [comment]
> The NECESSARY materials found to be required:
> 
> * Raspberry Pi
> * Plastic or Wood (Must look further into pricing)
> * Sautering Kit
> * Illuminance Meter
> 
> Other materials may be found to be required for the foreseeable future.

Another necessary item that we will need is a small motor, most likely multiple. We will need them in order to control the window shade by connecting them to the Raspberry Pi.

**Comment by HengYiLee** [comment]
> I think we should look into making out product an attachment rather than a permanent fixture.

I agree with this statement as many homeowners already have existing window shades, and therefore will likely not purchase a brand new window shade just for the automatic feature, while they could still use their old window shade.

**Comment by adheesh-k** [comment]
> > I think we should look into making out product an attachment rather than a permanent fixture.
> 
> I agree with this statement as many homeowners already have existing window shades, and therefore will likely not purchase a brand new window shade just for the automatic feature, while they could still use their old window shade.

I agree, we should do this in order to minimize costs and make it more appealing to consumers in comparison to already existing automated shades that are usually quite expensive.
